site-src/guides/migrating-from-ingress.md

```diff
@@ -0,0 +1,261 @@
+# Migrating from Ingress
+
+The Gateway API is not backward compatible with the [Ingress API](https://kubernetes.io/docs/concepts/services-networking/ingress/). As a result, if you are an Ingress user and would like to migrate, you will need to convert your Ingress resources to Gateway API resources. 
+
+This guide will help you with the conversion. It will:
+
+* Explain the key differences between the Ingress API and the Gateway API.
+* Map the Ingress features to the Gateway API features.
+* Show an example of an Ingress resource converted to Gateway API resources.
+* Mention a helper tool for automatic conversion.
+
+At the same time, it will not prepare you for a live migration or explain how to convert some vendor-specific features of your Ingress controller. Additionally, since the Ingress API only covers HTTP/HTTPS traffic, this guide does not cover the Gateway API support for other protocols.
+
+## Key Differences Between Ingress API and Gateway API
+
+There are three major differences between the Ingress API and the Gateway API:
+
+* Personas.
+* Available features.
+* Approach to extensibility (vendor-specific features).
+
+### Personas
+
+At first, the Ingress API had only a single resource kind Ingress. As a result, it had only one persona -- the user -- the owner of Ingress resources. The Ingress features give the user a lot of control over how applications are exposed to their external clients, including TLS termination configuration and provisioning of the load balancing infrastructure (supported by some Ingress controllers). Such a level of control is called the self-service model. 
```

Comment:
Tiny nits:

1. If you can wrap lines at 80 chars it makes it easier to leave comments/review smaller chunks at a time.
2. The first sentence feels just a bit off to me, maybe "Initially the Ingress API only had a single kind of resource."